ls1intum/Artemis
# Feature/overview/loading icon for views

### Checklist

### Motivation and Context


Closes #1337 and displays a loading notification when a HTTP Request takes longer than 1000ms.

### Description

- Created loading-notification component and service
- The loading notification will be displayed in the navigation bar next to the version
- We are using HttpInterceptor to trigger if the loading notification will be visible or not
### Steps for Testing


1. Log in to Artemis
2. Navigate through the application (views, create exercises, dashboards)
3. Check whether the loading spinner appears or not. To do so use [throttling](https://css-tricks.com/throttling-the-network/) to trigger the loading spinner
4. Test edge/weird cases (e.g. lost internet connection or changing views fast) and see if loading spinner disappears

### Screenshots


This is how the loading spinner should appear when the http requests take longer than 1000ms (in this gif, the time was set to 100ms in order to trigger the animation easier)
![video1](https://user-images.githubusercontent.com/41108487/84086825-0d125200-a9e9-11ea-8a5b-65c647c98236.gif)
